course6/toyapp_mikhail/displaylib/vector_common.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+namespace Display{
+
+template <class TValue, int TDimentions>
+class VectorCommon
+{
+	// No constructor for vector without specification
```

Comment:
Simply provide no definition for primary template at all, then only specializations will be allowed. 
But in this case I was expecting an actual primary template that works with any size